communityrust.properties

```diff
@@ -1,11 +1,18 @@
 category=Languages
 description=Code Analyzer for Rust
 homepageUrl=https://github.com/elegoff/sonar-rust
-publicVersions=0.0.6
+
+publicVersions=0.0.7
 
 defaults.mavenGroupId=org.elegoff
 defaults.mavenArtifactId=community-rust-plugin
 
+0.0.7.description=Update with latest Rust grammer and refeshed list of Clippy list 
+0.0.7.sqVersions=[7.9,LATEST]
+0.0.7.date=2021-11-25
+0.0.7.changelogUrl=https://github.com/elegoff/sonar-rust/releases/tag/v0.0.7
+0.0.7.downloadUrl=https://github.com/elegoff/sonar-rust/releases/download/v0.0.7/community-rust-plugin-0.0.7.jar
+
 0.0.6.description=Initial Marketplace entry
 0.0.6.sqVersions=[7.9,LATEST]
```

Comment:
Only one version can be marked compatible with `LATEST` at a time. This range needs to end with the current version. Since there's already a patch out for 9.2, I would just use `9.2.*`.